libs/newsletter-workflow/src/lib/steps/newsletterData/illustrationCardLayout.ts

```diff
@@ -0,0 +1,53 @@
+import type { DraftService } from '@newsletters-nx/newsletters-data-client';
+import { getNextStepId } from '@newsletters-nx/state-machine';
+import type { WizardStepLayout } from '@newsletters-nx/state-machine';
+import { executeModify } from '../../executeModify';
+import { executeSkip } from '../../executeSkip';
+import { getStringValuesFromRecord } from '../../getValuesFromRecord';
+import { regExPatterns } from '../../regExPatterns';
+import { formSchemas } from './formSchemas';
+
+const markdownTemplate = `
+## Add an illustration card for of {{name}}
+
+Illustration card images are used to promote newsletters on the newsletters page and on thrashers.
```

Comment:
We don't actually use the illustrationCard image in thrashers as such - we might use the same asset as the 'mobile' version of the graphic, but on most views it needs to be a square image. it might be best not to mentions thrashers here.

Reply:
Ahh cheers - misunderstood that use 👍